webclient/src/components/Toast/index.ts

```diff
@@ -0,0 +1,8 @@
+import { useToast, ToastProvider } from './ToastContext';
+import Toast from './Toast';
+
+export {
+  Toast as default,
```

Comment:
Toast is just an interface, lets just export it normal.

Reply:
Great catch!  This was from a previous iteration and is only meant for use within the component itself.

Do you think it's worth the effort to expose a `ToastHookOptions` for the `useToast` hook in it's place?

Reply:
For sure, more typing the better.